### PR TITLE
constrain image sequences to a single sample description entry and to…

### LINKED ISSUE #108: Image sequence vs. AV1 Sequences
The image sequence section (https://aomediacodec.github.io/av1-avif/#image-sequences) should be clarified regarding how multiple AV1 sequences (e.g. 2 sets of OBUs each starting with a Sequence Header OBU) should be treated. 
Options are:
1. they are not permitted. One pict track means one AV1 sequence, which means one Sequence Header OBU
2. they are permitted but each sequence should correspond to a single sample description entry: 1 sample description entry matches 1 Sequence Header OBU
3. they are permitted and can be mixed in the scope of one sample description entry as long as it does not violate the matching rules for the content of the av1C box.

### PR DESCRIPTION
**Pull request by cconcolato**
… constant sequence header

closes #108

***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/av1-avif/pull/148.html" title="Last updated on Apr 29, 2021, 7:10 PM UTC (df2b05a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/av1-avif/148/9f645f1...df2b05a.html" title="Last updated on Apr 29, 2021, 7:10 PM UTC (df2b05a)">Diff</a>

### CODE REVIEW

**Comment by joedrago** [comment]
Looks good to me.

File: index.bs [hunk 1]
```diff
@@ -313,6 +313,8 @@ NOTE: The size of the last layer can be determined by subtracting the sum of the
 
     - The track shall be a valid [=MIAF image sequence=].
     - The track handler for an [=AV1 Image Sequence=] shall be <code>pict</code>.
+    - The track shall have only one sample description entry.
+    - If multiple [=Sequence Header OBUs=] are present in the track payload, they shall be identical.
```
Comment (wantehchang) [inline index.bs:317]: The exact requirement in the AV1 specification (Section 7.5, page 205) is the following:

> Within a particular coded video sequence, the contents of sequence_header_obu must be bit-identical each time the sequence header appears except for the contents of operating_parameters_info.

To match the AV1 specification exactly, we can change "identical" to "bit-identical" and add "except for the contents of operating_parameters_info" to the end of this sentence. But I am not sure if we need to be that precise.

**Review by wantehchang — APPROVED** [review]
LGTM. Thanks!

**Review by leo-barnes — APPROVED** [review]
👍

**Pull request merged**